website/static/css/pages/wiki-page.css

```diff
@@ -328,3 +328,19 @@ select.form-control {
 .wiki-toolbar-icon:hover {
   background: rgba(51, 122, 183, 0.15);
 }
+
+.absolute-center-loading-ball{
```

Comment:
Is there anything specific to the loading indicator here? If not, perhaps this should be renamed to just `.absolute-center`.

Reply:
On line 343 we specify a size for .absolute-center-loading-ball's child div. I wouldn't want to generalize this, in case that's an unintended side effect.

Reply:
I see that, but this declaration doesn't have anything to do with the loading indicator. The declaration on L343 is specific to the Wiki editor's loading indicator, so maybe it makes more sense to do

```css
.wmd-panel .ball-scale.absolute-center > div {

}
```